webapp/tests/Unit/Controller/Jury/tmpTest.php

```diff
@@ -0,0 +1,52 @@
+webapp/tests/Unit/Controller/Jury/ContestControllerTest.php
```

Comment:
What’s this file?

Reply:
This branch originally also had tests... in refactoring I moved these to this tmp file, should not be needed in this form